UIKitDZ/UIKitDZ/Controllers/BMIViewController.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  BMIViewController.swift
+//  UIKitDZ
+//
+//  Created by 2lup on 03.10.2022.
+//
+
+import UIKit
+
+// MARK: - BMIViewController
+
+/// Экран с расчетом ИМТ человка
+final class BMIViewController: UIViewController {
+    
+    // MARK: - IBOutlet
+    
+    @IBOutlet private weak var predictionBMISegmented: UISegmentedControl!
+    @IBOutlet private weak var heightPersonLabel: UILabel!
+    @IBOutlet private weak var heightPersonSlider: UISlider!
+    @IBOutlet private weak var weightPersonLabel: UILabel!
+    @IBOutlet private weak var weightPersonPickerView: UIPickerView!
+    @IBOutlet private weak var resultBMILabel: UILabel!
+    
+    // MARK: - Private Properties
+    
+    private var personBMIData = PersonBMIData()
```

Comment:
```suggestion
    private var personBMIData = PersonBMIData()
```
нельзя добавлять Data если тип не Data